app/src/main/java/com/jerboa/ui/components/comment/CommentNode.kt

```diff
@@ -315,12 +316,17 @@ fun LazyListScope.commentNodeItem(
                                 CommentBody(
                                     comment = commentView.comment,
                                     viewSource = viewSource,
-                                    onClick = { onCommentClick(commentView) },
+                                    onClick = {
+                                        if (!viewSource) {
+                                            onCommentClick(commentView)
+                                        }
+                                    },
                                     onLongClick = {
-                                        toggleActionBar(commentId)
+                                        if (!viewSource) {
+                                            toggleActionBar(commentId)
+                                        }
                                     },
                                 )
-                                Spacer(modifier = Modifier.height(MEDIUM_PADDING))
```

Comment:
The combination of this unclickable spacer and the modifier with unclickable padding on the footer line below seems to have been a deadzone when clicking to collapse comments.